entities/markets/market.ts

```diff
@@ -68,8 +69,24 @@ export class Market {
   }
 
   getWinnerOutcome() {
-    return this.isClosed && this.answer !== null && this.answer !== Market.INVALID_ANSWER
+    return this.isClosed && this.answer !== null && !this.isAnswerInvalid
       ? this.outcomes[this.answer]
       : null;
   }
+
+  getCurrentAnswerOutcome() {
+    return this.currentAnswer !== null && !this.isCurrentAnswerInvalid
+      ? this.outcomes[this.currentAnswer]
+      : null;
```

Comment:
why we're checking if `isCurrentAnswerInvalid`, wouldn't we also expect to receive invalid if it is the case on `getCurrentAnswerOutcome()`?

Reply:
not sure I understood

Reply:
If answer is invalid we are giving it a -1 index. We can't get -1 index from outcomes array to represent an invalid outcome. Currently we are returning null. There is a possibility of using `.at` method to return `array.length - 1 ` outcome.

Reply:
yeah, we can try that

Reply:
Done